src/blocks/sticky-menu/view.js

```
@@ -0,0 +1,625 @@
+	// Add keyboard event listener for Enter, Space, and Escape keys
+	existing_header.addEventListener('keydown', function(event) {
+		switch(event.key) {
+			case 'Enter':
+			case ' ':
+				event.preventDefault();
+				lsx_to.toggle_mobile_section(wrapper);
+				break;
+			case 'Escape':
+				event.preventDefault();
+				// Always collapse on Escape if expanded
+				const is_expanded = wrapper.getAttribute('aria-expanded') === 'true';
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)